src/main/java/no/nav/finnkandidatapi/midlertidigutilgjengelig/MidlertidigUtilgjengeligOutboundDto.java

```diff
@@ -0,0 +1,13 @@
+package no.nav.finnkandidatapi.midlertidigutilgjengelig;
+
+public class MidlertidigUtilgjengeligOutboundDto {
```

Comment:
Litt kjipt å måtte dele opp i inbound og outbound synes jeg, i og med at klassene er såpass like 😕 Jeg hadde håpet det var en Spring- eller Jackson-annotasjon som bare fikset denne wrappingen for oss, men det ser ut til at vi i så fall hadde måttet lage det selv.

Anyway, hvis vi bruker outbound-objektet i get-endepunktet, så burde vi bruke det i post og put også, for å få konsekvent håndtering av responsen i frontend

Reply:
> Litt kjipt å måtte dele opp i inbound og outbound synes jeg, i og med at klassene er såpass like 😕 Jeg hadde håpet det var en Spring- eller Jackson-annotasjon som bare fikset denne wrappingen for oss, men det ser ut til at vi i så fall hadde måttet lage det selv.

Skjønner ikke; klassene MidlertidigUtilgjengeligOutboundDto og MidlertidigUtilgjengeligInboundDto er vel ikke like?

Reply:
>Anyway, hvis vi bruker outbound-objektet i get-endepunktet, så burde vi bruke det i post og put også, for å få konsekvent håndtering av responsen i frontend
Enig. Skal jeg gjøre det med det samme, @kjesvale?

Reply:
PUT og POST bruker nå samme nye DTO som GET

Reply:
> > Litt kjipt å måtte dele opp i inbound og outbound synes jeg, i og med at klassene er såpass like 😕 Jeg hadde håpet det var en Spring- eller Jackson-annotasjon som bare fikset denne wrappingen for oss, men det ser ut til at vi i så fall hadde måttet lage det selv.
> 
> Skjønner ikke; klassene MidlertidigUtilgjengeligOutboundDto og MidlertidigUtilgjengeligInboundDto er vel ikke like?

Nei, har du sett, det er de jo ikke 😮 Der blingsa jeg litt, sorry 😅

Reply:
Gode gamle Blings kjenner jeg godt ;)